src/main/java/uk/gov/register/util/HashValue.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.register.util;
+
+import uk.gov.register.core.HashingAlgorithm;
+
+public class HashValue {
+    private final String rawHash;
+    private final String encodedHash;
+
+    public HashValue(HashingAlgorithm hashingAlgorithm, String rawHash) {
+        this.rawHash = rawHash;
+        this.encodedHash = hashingAlgorithm.toString() + ":" + rawHash;
+    }
```

Comment:
We don't need to store the encoded hash but compute it on the fly in the encode() method?